src/components/WorkMetadata/WorkMetadata.module.scss

```
@@ -2,6 +2,11 @@
 	color: var(--datacite-color-primary, #88939a) !important;
 }
 
-.id {
-	color: #3498db !important;
+a.id {
+	color: var(--datacite-color-secondary, #3498db) !important;
+	// color: yellow;
```

Stylelint (17.9.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)